src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -2,9 +2,10 @@ import React, { useState } from "react";
 import PropTypes from "prop-types";
 import { sanitizeText } from "utils/helpers/sanitizeText.js";
 import { KEY_ENTER } from "utils/constant/form";
+import DeleteIcon from "components/Shared/Image/DeleteIcon";
```

Comment:
directory names should be lowercase.

```suggestion
import DeleteIcon from "components/shared/DeleteIcon";
```